### PR TITLE
centiloid pipeline

### PR DESCRIPTION
**Pull request by casperdcl**
- add `centiloid` pipeline
  + add CSV output (#32)
- misc bug fixes
  + fix windows install (fix #30)
- [ ] address review comments
- [ ] bump dependency versions
  + [x] https://github.com/AMYPAD/NiftyPAD/pull/6
  + [x] https://github.com/NiftyPET/NIMPA/pull/28
  + [x] https://github.com/AMYPAD/SPM12/pull/19
    * [x] https://github.com/AMYPAD/miutil/pull/32

### CODE REVIEW

File: amypet/preproc.py [hunk 1]
```diff
@@ -50,13 +52,106 @@
 # ------------------------------------------------
 
 
+# ========================================================================================
+
+def dicom2nifti(inpath, outpath=None, ignore_derived=True, remove_previous=False):
```
Comment (casperdcl) [inline amypet/preproc.py:57]: @pjmark what about the different approach in the `dcm2nii.py` file (relies on `nimpa.dcm2nii`)?

File: amypet/utils.py [hunk 2]
```diff
@@ -7,3 +21,665 @@
...
+
+        # > atlas dictionary
+        datlas = dict((i[0].text, i[1].text) for i in lbls)
+
+        outdct = dict(fatlas=fatl, flabels=flbl, vois=datlas)
+
+
+    elif atlas.lower()=='hammers':
+        
+        # > main URL
```
Comment (casperdcl) [inline amypet/utils.py:72]: should use `miutil.web` and `miutil.fdio` for this?

File: amypet/preproc.py (hunk 1 above)
Comment (pjmark) [reply amypet/preproc.py:57]: pretty open, as long dcm2niix is used

File: amypet/utils.py (hunk 2 above)
Comment (pjmark) [reply amypet/utils.py:72]: if it gets the atlas, it can be anything.

**Review by pjmark — APPROVED** [review]

**Pull request closed without merging**